groupaccess.c

```diff
@@ -49,6 +52,73 @@ static char **groups_byname;
 int
 ga_init(const char *user, gid_t base)
 {
+#ifdef WINDOWS
+	DWORD i = 0, j = 0;
+	LPLOCALGROUP_USERS_INFO_0 local_groups_info = NULL, tmp_groups_info;
+	wchar_t *user_utf16 = NULL, *full_name_utf16 = NULL, *udom_utf16 = NULL, *tmp;
+	DWORD entries_read = 0, total_entries = 0, full_name_len = 0, index = 0;
+	NET_API_STATUS nStatus;
+	
+	if (ngroups > 0)
+		ga_free();
+
+	user_utf16 = utf8_to_utf16(user);
+	full_name_len = wcslen(user_utf16) + 1;
+	if ((full_name_utf16 = malloc(full_name_len * sizeof(wchar_t))) == NULL) {
+		errno = ENOMEM;
+		goto done;
+	}
+	
+	if ((tmp = wcschr(user_utf16, L'@')) != NULL) {
+		udom_utf16 = tmp + 1;
+		*tmp = L'\0';
+		index = wcslen(udom_utf16) + 1;
+		wmemcpy(full_name_utf16, udom_utf16, index);
+		full_name_utf16[wcslen(udom_utf16)] = L'\\';
+	}
+	wmemcpy(full_name_utf16 + index, user_utf16, wcslen(user_utf16) + 1);
+
+	nStatus = NetUserGetLocalGroups(NULL,
```

Comment:
No, only local groups are returned. but flag LG_INCLUDE_INDIRECT will guarantee the group still returned if the user belong to some other global group and the global group belong to the local group.